resources/views/vendor/backpack/base/inc/sidebar_content.blade.php

```diff
@@ -53,6 +53,7 @@
   <ul class="nav-dropdown-items">
       <li class="nav-item"><a class="nav-link" href="{{ backpack_url('icon') }}"><i class="nav-icon la la-info-circle"></i> <span>Icons</span></a></li>
       <li class="nav-item"><a class="nav-link" href="{{ backpack_url('product') }}"><i class="nav-icon la la-shopping-cart"></i> <span>Products</span></a></li>
+      <li class="nav-item"><a class="nav-link" href="{{ backpack_url('column-monster') }}"><i class="nav-icon la la-list-alt"></i> Column monsters</a></li>
```

Comment:
Opinionated change - I just like all sidebar items to have different icons, that way they're easier to differentiate:

```suggestion
      <li class="nav-item"><a class="nav-link" href="{{ backpack_url('column-monster') }}"><i class="nav-icon la la-table"></i> Column monsters</a></li>
```